### PR TITLE
Remove size in OrderedSet

### PR DESCRIPTION
**Pull request by weiihann**

### CODE REVIEW

**Review by kirugan — APPROVED** [review]

**Review by AnkushinDaniil — APPROVED** [review]

File: utils/orderedset.go [hunk 1]
```diff
@@ -22,46 +21,45 @@ func NewOrderedSet[K comparable, V any]() *OrderedSet[K, V] {
 	}
 }
 
-func (ps *OrderedSet[K, V]) Put(key K, value V) {
-	ps.lock.Lock()
-	defer ps.lock.Unlock()
+func (o *OrderedSet[K, V]) Put(key K, value V) {
```
Comment (pnowosie) [inline utils/orderedset.go:24]: One tiny extension, let it return `bool` value indicating that value was new. 
Returns `True` if the element was added to the set; `False` if the element was already present.

```suggestion
func (o *OrderedSet[K, V]) Put(key K, value V) bool {
```

**Review by pnowosie — APPROVED** [review]
👍 Nice

File: utils/orderedset.go (hunk 1 above)
Comment (pnowosie) [reply utils/orderedset.go:24]: Or on the 2nd thought we might return the previous value on update (?) 
WDYT? 🤔

Comment (weiihann) [reply utils/orderedset.go:24]: This is a valid suggestion, but we have no use case of returning `bool`/`V` for now. So I'll leave it to future update if we ever need it.

**Pull request merged**